File: 백준/Bronze/25304. 영수증/영수증.cc
#include <iostream>

int main(){
    int a, b, x, n, sum = 0;
    scanf("%d", &x);
    scanf("%d", &n);
    for(int i = 1; i <= n; i++){
        scanf("%d %d", &a, &b);
        sum += (a*b);
    }
    if(sum == x) printf("Yes");
    else printf("No");
}

File: 백준/Bronze/25304. 영수증/영수증.swift
let totalCost = Int(readLine()!)!
let products = Int(readLine()!)!
var sum = 0

for i in 1...products {
    let input = readLine()!.split(separator: " ").map{ Int($0)! }
    
    sum += (input[0] * input[1])
}

if sum == totalCost {
    print("Yes")
} else {
    print("No")
}